db/schema.rb

```diff
@@ -60,6 +60,7 @@
     t.string "address2"
     t.string "postcode"
     t.string "phonenumber"
+    t.integer "imdb_id"
```

Comment:
@akilchomoko why does a user have an IMDB id?

Reply:
It shouldn't. There should have been a migration that removed it from an earlier mistake.